subsys/spm/Kconfig

```diff
@@ -110,7 +110,7 @@ config SPM_SERVICE_FIND_FIRMWARE_INFO
 
 config SPM_SERVICE_S0_ACTIVE
 	bool "Enable secure service to check if S0 is active B1 slot"
-	default y
+	default n
```

Comment:
Unless there is a `default y` earlier you want to overrule, then `default n` is not needed.
```suggestion
```

Reply:
I know, I was following the convention in this file, I will remove it.